cypress/support/commands.js

```diff
@@ -32,10 +32,31 @@ Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+Cypress.Commands.add('register', ({ email, username, password }) => {
   cy.request('POST', '/users', {
     email,
     username,
     password
-  });
+  }).then((res) => ({
+    ...res.body.user,
+    password
+  }));
+});
```

Comment:
The 'register' command does not handle errors, which could lead to unhandled promise rejections if the registration fails. Consider adding error handling to manage potential failures gracefully.